src/main/java/school/lemon/changerequest/java/introduction/hw2/Task7.java

```diff
@@ -2,6 +2,13 @@
 
 public class Task7 {
     public static int countEvens(int[] arr) {
-        return 0;
+
+        if(arr == null)
+            return 0;
+        int counter = 0;
+        for(int a: arr)
+            if(a%2 == 0)
```

Comment:
There is more efficient way to check whether number is even or odd - [parity bit check](https://en.wikipedia.org/wiki/Parity_bit)